test/test_set_failures_to_zero.py

```diff
@@ -0,0 +1,69 @@
+import gw_eccentricity
+from gw_eccentricity import load_data
+from gw_eccentricity import measure_eccentricity
+import numpy as np
+
+
+def test_set_failures_to_zero():
+    """Test handling of failures due to insufficient extrema.
+
+    In certain situations, the waveform may have zero eccentricity or a very
+    small eccentricity, making it difficult for the given method to identify
+    any extrema. In cases where such a situation occurs, and if the user has
+    set 'set_failures_to_zero' to `True` in the 'extra_kwargs' parameter, both
+    the eccentricity and mean anomaly will be set to zero.
```

Comment:
This doesn't say what this is testing: it's testing that the interface works with set_failures_to_zero for waveforms with small or zero ecc.

Reply:
Done